subgraph/subgraph.template.yaml

```diff
@@ -74,3 +74,28 @@ dataSources:
           handler: handleRearrangeDeposit
         - event: RearrangeRedeemOperation(indexed address,indexed address,indexed uint256)
           handler: handleRearrangeRedeem
+
+  - name: Donations
+    kind: ethereum/contract
+    network: {{network}}
```

Comment:
Will this work when Donations are in Polygon?

Reply:
I think you're right this needs to be a seperate subgraph because not the same network

didn't consider this :sob: